composer.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "JetBrains/phpstorm-stubs",
```

Comment:
There is another JetBrains package published on packagist though it use all lower case for the company name: https://github.com/JetBrains/phpstorm-workshop/blob/master/composer.json done by @maartenba

    "name": "jetbrains/phpstorm-workshop"

Might want to stay consistent?